### DISCUSSION TITLE
zh/guide/drivers/Alist%20V2%20V3

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#145] [Announcements]
# zh/guide/drivers/Alist%20V2%20V3

alist 驱动 首先你要知道对方的到底是V2还是V3 链接 你要挂载的Alist链接 根目录路径 要添加的目录的pathname，不填默认为根目录，如：; https://alist.nn.ci (https://alist.nn.ci/) -> /; https://alist.nn.ci/a -> /a; https://alist.nn.ci/...

https://alist.nn.ci/zh/guide/drivers/Alist%20V2%20V3.html

**Comment by ZestfulCitrus** [comment]
有个问题，在我挂载了其他alist后，文件无法上传

**Reply by ZestfulCitrus** [reply]
搞明白了，https导致的，我再申请个证书就好了。

**Comment by yaorelax** [comment]
如果挂载的alist需要用户名和密码，怎么操作呢？

**Comment by yingxianwang** [comment]
failed get objs: failed to list objs: Post "https://www.asmrgay.com/api/fs/list": read tcp ***********:51777->***********00:443: wsarecv: An existing connection was forcibly closed by the remote host.

这什么情况

**Reply by ZhengZhaoxi** [reply]
你解决这个问题了吗

**Comment by viosonlee** [comment]
说实话，硬是没有找到输入用户名的地方

**Comment by at86397388** [comment]
允许或者不允许别人挂载怎么设置呢？

**Reply by jiabingxian** [reply]
设置——站点——允许挂载

**Comment by ZhengZhaoxi** [comment]
failed get objs: failed to list objs: Post "https://pan.clun.top/api/public/path": read tcp [**************************************]:11144->[*************************]:443: wsarecv: An existing connection was forcibly closed by the remote host.这个报错怎么搞

**Comment by taotaozuiqiang** [comment]
如果对方开了二次身份认证怎么设置

**Reply by zuilintan** [reply]
那个无所谓, 拿到对方
-设置
-其它
-令牌
即可

**Comment by kzcy** [comment]
Failed load storage: failed init storage: parse "************15:5665/api/me": first path segment in URL cannot contain colon

**Comment by jessecheng4869** [comment]
Failed init storage: request failed,code: 0, message:提示这个是什么意思

**Comment by ypq123456789** [comment]
总是令牌过期怎么弄

**Reply by ypq123456789** [reply]
重新保存就可以了，但是总是手动太傻了，而且令牌过期非常快

**Comment by Haruka-24** [comment]
我挂载了之后没有内容是怎么回事？网页上可以看到内容的，但是挂载之后就不行了。

**Comment by 13342325377** [comment]
非常好👍
